MeetingNMusae/MeetingNMusae/Views/MissionCardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  MissionCardView.swift
+//  MeetingNMusae
+//
+//  Created by Hyorim Nam on 2022/06/11.
+//
+
+import SwiftUI
+
+struct MissionCardView: View {
+    // 파이어베이스 연결할 때 프로퍼티 래퍼 사용할 것
+    @State var role = "금고무새"
+    @State var didMissions: [Bool] = [true, false, false]
+
+    var body: some View {
+        // 서버에서 받을 데이터: 현재 역할, 미션, 미션 수행 했는지
+        // 서버에 보낼 데이터: 미션 수행 했는지
+        let roleName = role
+        let imageName = role
+        let missions = ["미션1", "미션2", "미션3"]
+
+        VStack {
+            Image(imageName)
+                .resizable()
+                .scaledToFit()
+            Text(roleName)
+
+            // will be changed to a custom line - based on 역할 선택 (상세) view
+            Divider()
```

Comment:
```
//
//  Line.swift
//  MeetingNMusae
//
//  Created by taekkim on 2022/06/11.
//

import SwiftUI

struct Line: Shape { func path(in rect: CGRect) -> Path {
        var path = Path()
        path.move(to: CGPoint(x: 0, y: 0))
        path.addLine(to: CGPoint(x: rect.width, y: 0))
        return path
    }
}
```
Views에 Line 이라는 파일에 Line struct 만들어놨습니다


```suggestion
            Line()
                .stroke(style: StrokeStyle(lineWidth: 3, dash: [10]))
                .frame(height: 1)
```

요롷게 쓰시면 될 거 같아여!!!

Reply:
감사합니다. 수정했습니다!
0d4acf84dbeeb11e3bbaf99619a27f9c1fb6b62d